.repo/sample-list-generator/extension-apis.json

```diff
@@ -0,0 +1,1994 @@
+{
```

Comment:
Can you add a flag to this file indicating that this is auto generated and should not be edited?

Reply:
This sounds good but I don't think we should do a comment since only certain JSON parsers support that. Are you thinking something like this?

```json
{
  "_comment": "This file is autogenerated. Do not edit.",
  ...
}
```